proto/src/domain/notes.rs

```diff
@@ -2,21 +2,6 @@ use miden_objects::notes::NoteEnvelope;
 
 use crate::generated::note;
 
-// Note
-// ================================================================================================
-
-impl From<note::Note> for note::NoteSyncRecord {
-    fn from(value: note::Note) -> Self {
-        Self {
-            note_index: value.note_index,
-            note_id: value.note_id,
-            sender: value.sender,
-            tag: value.tag,
-            merkle_path: value.merkle_path,
-        }
-    }
-}
```

Comment:
Question: why do we no longer need this?

Reply:
I was just trying to make sure that we were not setting the incorrect `note_index` and checked all references which brought me to this. I deleted it to see if it would bring up any place where we were using this and saw that it was not being used at all. I can bring it back.

Reply:
If it is not used anywhere - no need to bring it back.

Reply:
Agreed, that's what I thought